app/jinja_filters.py

```diff
@@ -67,7 +68,9 @@ def format_multilined_string(context, value):
 
 @blueprint.app_template_filter()
 def format_date(value, date_format='%-d %B %Y'):
-    return "<span class='date'>{date}</span>".format(date=datetime.strptime(value, '%Y-%m-%d').strftime(date_format))
+    if value and len(value) == 7:
```

Comment:
I would probably use a regex rather then count the chars

Reply:
Reused logic from the `convert_to_datetime` method. Shall modify that method to match?